streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -741,7 +741,7 @@ int commitAll() {
         return commitInternal(tasks.values());
     }
 
-    private int commitInternal(final Collection<Task> tasks) {
+    int commitInternal(final Collection<Task> tasks) {
```

Comment:
Should we consider adding a unit test here, since this call is externalized?

Reply:
Good call.

I also need to add a unit test that we actually commit all other tasks if a `TaskCorruptedException` is thrown. Just wanted to get the suggested fix reviewed first (also tested via system test run) before I close the unit test gaps.